modules/gapi/cmake/init.cmake

```diff
@@ -32,3 +32,8 @@ if(WITH_PLAIDML)
       set(HAVE_PLAIDML TRUE)
   endif()
 endif()
+
+if(WITH_GAPI_ONEVPL)
```

Comment:
Should it be `WITH_GAPI_` or just `WITH_`? The above two options don't have `GAPI` in the name.. @alalek do any other modules react to `WITH_VPL`?